nasl/CMakeLists.txt

```diff
@@ -37,44 +37,53 @@ pkg_check_modules (OPENVAS_WMICLIENT libopenvas_wmiclient>=1.0.5)
 # for 'nasl' binary
 pkg_check_modules (LIBSSH REQUIRED libssh>=0.6.0)
 
+
 set (KSBA_MIN_VERSION "1.0.7")
-message (STATUS "Looking for ksba >= ${KSBA_MIN_VERSION}...")
-find_library (KSBA ksba)
-message (STATUS "Looking for ksba >= ${KSBA_MIN_VERSION}... ${KSBA}")
-if (NOT KSBA)
-  message (SEND_ERROR "The ksba library is required.")
-else (NOT KSBA)
-  execute_process (COMMAND ksba-config --version
-    OUTPUT_VARIABLE KSBA_VERSION
-    OUTPUT_STRIP_TRAILING_WHITESPACE)
-  message (STATUS "Found ksba ${KSBA_VERSION}...")
-  if (${KSBA_VERSION} VERSION_LESS ${KSBA_MIN_VERSION})
-    message (SEND_ERROR "The ksba library >= ${KSBA_MIN_VERSION} is required.")
-  else (${KSBA_VERSION} VERSION_LESS ${KSBA_MIN_VERSION})
-    execute_process (COMMAND ksba-config --libs
-      OUTPUT_VARIABLE KSBA_LDFLAGS
-      OUTPUT_STRIP_TRAILING_WHITESPACE)
-    execute_process (COMMAND ksba-config --cflags
-      OUTPUT_VARIABLE KSBA_CFLAGS
+# try with package config
+pkg_check_modules (KSBA REQUIRED ksba>=1.0.7)
+if (NOT KSBA_FOUND)
+  # try with find library
+  find_library (KSBA ksba)
+  message (STATUS "Looking for ksba >= ${KSBA_MIN_VERSION}...")
+  if (NOT KSBA)
+    message (SEND_ERROR "The ksba library is required.")
+  else (NOT KSBA)
+    execute_process (COMMAND ksba-config --version
+      OUTPUT_VARIABLE KSBA_VERSION
       OUTPUT_STRIP_TRAILING_WHITESPACE)
-  endif (${KSBA_VERSION} VERSION_LESS ${KSBA_MIN_VERSION})
-endif (NOT KSBA)
+    message (STATUS "Found ksba ${KSBA_VERSION}...")
+    if (${KSBA_VERSION} VERSION_LESS ${KSBA_MIN_VERSION})
+      message (SEND_ERROR "The ksba library >= ${KSBA_MIN_VERSION} is required.")
+    else (${KSBA_VERSION} VERSION_LESS ${KSBA_MIN_VERSION})
+      execute_process (COMMAND ksba-config --libs
+        OUTPUT_VARIABLE KSBA_LDFLAGS
+        OUTPUT_STRIP_TRAILING_WHITESPACE)
+      execute_process (COMMAND ksba-config --cflags
+        OUTPUT_VARIABLE KSBA_CFLAGS
+        OUTPUT_STRIP_TRAILING_WHITESPACE)
+    endif (${KSBA_VERSION} VERSION_LESS ${KSBA_MIN_VERSION})
+  endif (NOT KSBA)
+endif (NOT KSBA_FOUND)
 
 message (STATUS "Looking for pcap...")
 find_library (PCAP pcap)
 
-find_library (GPGME gpgme)
-message (STATUS "Looking for gpgme... ${GPGME}")
-if (NOT GPGME)
-  message (SEND_ERROR "The gpgme library is required.")
-endif (NOT GPGME)
-
-execute_process (COMMAND gpgme-config --libs
-  OUTPUT_VARIABLE GPGME_LDFLAGS
-  OUTPUT_STRIP_TRAILING_WHITESPACE)
-execute_process (COMMAND gpgme-config --cflags
-  OUTPUT_VARIABLE GPGME_CFLAGS
-  OUTPUT_STRIP_TRAILING_WHITESPACE)
+pkg_check_modules (GPGME REQUIRED gpgme)
+if (GPGME_FOUND)
```

Comment:
I think that could also be `if (NOT GPGME_FOUND)`?

Reply:
:smiling_face_with_tear: